kalite/contentload/management/commands/generate_assessment_zips.py

```diff
@@ -0,0 +1,95 @@
+import copy
+import json
+import re
+import requests
+import os
+import zipfile
+from multiprocessing.dummy import Pool as ThreadPool
+
+from django.conf import settings
+from django.core.management.base import NoArgsCommand
+
+logging = settings.LOG
+
+ZIP_FILE_PATH = os.path.join(settings.PROJECT_PATH, "assessment_item_resources.zip")
+
+
+class Command(NoArgsCommand):
```

Comment:
I've judiciously used mocks and stubs in the tests I wrote. @MCGallaspy @anuragkanungo can you evaluate my usage in this file to see if this is what we want to do moving forward?